core/api/src/app/prices/display-currency-from-country-code.ts

```diff
@@ -0,0 +1,15 @@
+import { PriceService } from "@/services/price"
+
+export const displayCurrencyFromCountryCode = async (
```

Comment:
I implemented this at app layer, but is it something that could/should live in price service instead?

Reply:
Moved to domain https://github.com/GaloyMoney/galoy/pull/4309/commits/fccc7cf45bc8a4df664f38bd6ce81ecf49bfd8df